katas/metal-slug/jeanpeare/Vehicles.js

```diff
@@ -0,0 +1,6 @@
+export class Vehicles {
+    constructor(VehiclesType, Health){
+        this.VehiclesType = VehiclesType;
```

Comment:
Please use all lowercase for variables and attributes

Reply:
I will have in mind for future projects, because it is difficult change all the scripts, i'm sorry for that.

Reply:
Ok, no problem for this one, but for the next PRs we need to enforce the rule so it keeps consistency with the rest of the code. Thanks!